src/cmd-koji-upload

```diff
@@ -654,14 +650,11 @@ class Upload(_KojiBase):
                     }
                 },
                 "name": f"{self.build.build_name}-{self.build.basearch}",
-                "release": self._release,
+                "release": f"{self.build.build_id[-1:]}",
                 "owner": self._owner,
                 "source": source['origin'],
                 "start_time": stamp,
-                # RHCOS wants to be semver-compatible, but Koji doesn't
-                # accept `-`.  See
-                # https://github.com/openshift/oc/pull/209#issuecomment-564876535
-                "version": f"{self.build.build_id.replace('-', '.')}"
+                "version": f"{self.build.build_id[:-2]}"
```

Comment:
Need to do something similar to https://github.com/coreos/coreos-assembler/pull/3533#discussion_r1265821396 here